buf.yaml

```diff
@@ -0,0 +1,29 @@
+# buf.yaml files should always be in the root of your repository.
+build:
+  roots:
+    # Roots is the directories that all .proto files are contained within, and
+    # all imports must derive from.
+    # Only *.proto files within these directory will be compiled.
+    - .
+  # Excludes is the list of directories within root to exclude.
+  excludes:
+    - model/build
+
+# Lint contains the options for lint checks.
+lint:
+  # Use is the list of checker categories and ids to use for buf check lint.
+  use:
+    # Three category {@link 'https://buf.build/docs/lint-checkers'
+    # The BASIC category includes everything from the MINIMAL category, as well as the STYLE_BASIC category.
+    # The MINIMAL category represents what we consider to be fundamental rules for modern Protobuf development,
+    # regardless of style
+    # The DEFAULT category includes everything from the BASIC category, as well as the STYLE_DEFAULT category
+    - BASIC
+    # Checks that filenames are lower_snake_case.
+    - FILE_LOWER_SNAKE_CASE
+  # This removes ids or categories from the use list.
+  except:
+    # Checks that enums do not have the allow_alias option set.
+    - ENUM_NO_ALLOW_ALIAS
```

Comment:
As per the discussion, lets remove this line, and add the expected prefix/suffix into our enums. Feel free to make the changes to the enums in this PR itself (I think its only 2 enums in exploration.proto and analytics.proto as far as I remember)